DS-3136 Add arg to switch off call stack inspection in Sum*

jrwishart opened the pull request.

mwmclean commented:
Need to document `call.` and `roxygenize`.

I'm seeing one failed unit test
![image](https://user-images.githubusercontent.com/5276419/107309345-5c5fd380-6ade-11eb-91d9-29b38fa39ee3.png)

mwmclean commented:
I turned on `coveralls.io` for this repo, so should get coverage stats with the next successful build.

jrwishart commented:
> Need to document `call.` and `roxygenize`.
> 
> I'm seeing one failed unit test on reusdev
> ![image](https://user-images.githubusercontent.com/5276419/107309345-5c5fd380-6ade-11eb-91d9-29b38fa39ee3.png)

Updated the unit test to work on the Displayr servers. That test assumed the error message would be the general user error message and not the one for Displayr users. (hence the pass locally and on travis). Also updated the docs in the previous commit.
